metrics/src/test/java/com/expedia/adaptivealerting/metrics/functions/source/graphite/ConstructSourceURITest.java

```diff
@@ -6,15 +6,17 @@
 import com.typesafe.config.Config;
 import lombok.val;
 import org.junit.Test;
+
 import static org.junit.Assert.assertEquals;
 
 public class ConstructSourceURITest {
 
     @Test
     public void testGetGraphiteURI() {
-        val functionsInputFileName = "/config/functions-test.txt";
+        val functionsInputFileName = "config/functions-test.txt";
         val uri = "samplegraphitehosturi/render?until=now&format=json&target=sumSeries(a.b.c)&from=-30s";
-        MetricFunctionsSpec metricFunctionsSpec = MetricFunctionsReader.readFromInputFile(functionsInputFileName).get(0);
+        MetricFunctionsSpec metricFunctionsSpec = MetricFunctionsReader.
+                readFromInputFile(ClassLoader.getSystemResource(functionsInputFileName).getPath()).get(0);
```

Comment:
same as above

Reply:
Deployment uses absolute path of file - "/config/functions.txt". 
Updated test cases to reflect the same as well.